resident-services/scripts/Resident_Test_script.jmx

```diff
@@ -2363,7 +2363,7 @@ vars.put(&quot;utcTime&quot;, utcTime);</stringProp>
             <stringProp name="filename"></stringProp>
             <stringProp name="parameters"></stringProp>
             <boolProp name="resetInterpreter">false</boolProp>
-            <stringProp name="script">${__StringToFile(resident_revoke_vid.txt,${adminIdToken}\,${accessToken}\,${vidGen}\n,true,)}</stringProp>
+            <stringProp name="script">${__StringToFile(resident_revoke_vid.txt,${idToken}\,${accessToken}\,${vidGen}\n,true,)}</stringProp>
```

Comment:
Extract Event Id - why is event_id not camel case ?

Reply:
Done